### PR TITLE
Create session cookie

### PR DESCRIPTION
**Pull request by crobin27**
After researching how to best store the user's email data in the website, I decided to go with the session storage option. 
Although not a cookie, the session storage option is similar and was easier to use. 
See: https://www.digitalocean.com/community/tutorials/how-to-add-login-authentication-to-react-applications

The addition of the useLoginToken.js file makes it possible to use the token across all .js pages. 
Since the token is stored in session storage, the user's email will still be available even after a page refresh. 

After this addition, we will be able to customize the myShows page for the currently logged in user. Let me know if you have any suggestions or clarifications.

### CODE REVIEW

File: react-frontend/src/App.jsx [hunk 1]
```diff
@@ -1,20 +1,22 @@
-import React from 'react';
```
Comment (SamuelSehnert) [inline react-frontend/src/App.jsx:1]: Was this meant to be deleted? Isn't this required to render a react page?

**Review by SamuelSehnert — COMMENTED** [review]
LGTM, but I just want to make sure my comment in App.jsx is addressed! Great work!

**Review by GoScout2022 — APPROVED** [review]

**Review by mpicazocp — APPROVED** [review]
LGTM

**Comment by SamuelSehnert** [comment]
LGTM

**Pull request merged**